cat/CaT-service.yaml

```diff
@@ -1302,7 +1300,7 @@ paths:
       description: Get Event's for this Project. In OCDS this is the related cases of Type
       tags:
         - Manage Procurement Project
-      operationId: caseRFXs
+      operationId: RFXs
```

Comment:
Really trivial comment (feel free to ignore). Would this not be 'Get RFxs' as it is the name of the operation not the object. Even more trivial, small 'x'.
Just realised, operation names is one thing not covered by the standards. e.g. camel case, snake case?

Reply:
Hi, I think I have made this change as required. I think I am just blocked on you approving this. Should be in last update on this branch. If I have done it right approve and I will then merge.

Thanks